code/modules/mob/living/simple_animal/hostile/terror_spiders/black.dm

```diff
@@ -0,0 +1,64 @@
+
+// --------------------------------------------------------------------------------
+// ----------------- TERROR SPIDERS: T2 BLACK TERROR ------------------------------
+// --------------------------------------------------------------------------------
+// -------------: ROLE: assassin, poisoner, DoT expert
+// -------------: AI: attacks to inject its venom, then retreats. Will inject its enemies multiple times then hang back to ensure they die.
+// -------------: SPECIAL: venom that does more damage the more of it is in you
+// -------------: TO FIGHT IT: if bitten once, retreat, get charcoal/etc treatment, and come back with a gun.
+// -------------: CONCEPT: http://tvtropes.org/pmwiki/pmwiki.php/Main/GradualGrinder
+// -------------: SPRITES FROM: Travelling Merchant, http://nanotrasen.se/phpBB3/memberlist.php?mode=viewprofile&u=2766
+
+/mob/living/simple_animal/hostile/poison/terror_spider/black
+	name = "Black Terror spider"
+	desc = "An ominous-looking spider, black as the darkest night, and with merciless yellow eyes."
+	altnames = list("Black Devil spider", "Giant Black Widow spider", "Shadow Terror spider")
+	spider_role_summary = "Hit-and-run attacker with extremely venomous bite."
+	ai_target_method = TS_DAMAGE_POISON
+	egg_name = "black spider eggs"
+
+	icon_state = "terror_black"
+	icon_living = "terror_black"
+	icon_dead = "terror_black_dead"
+	maxHealth = 120 // same health as hunter spider, aka, pretty weak.. but its bite will kill you!
+	health = 120
+	melee_damage_lower = 5
+	melee_damage_upper = 15
+	move_to_delay = 5
+	stat_attack = 1 // ensures they will target people in crit, too!
+	spider_tier = 2
+
+
+/mob/living/simple_animal/hostile/poison/terror_spider/black/harvest()
+	new /obj/item/weapon/reagent_containers/terrorspider_parts/toxgland_black(get_turf(src))
+	gib()
+
+/mob/living/simple_animal/hostile/poison/terror_spider/black/ShowGuide()
+	..()
+	to_chat(src, "BLACK TERROR guide:")
```

Comment:
Didn't you just ask Kyep on Discord if you could keep line commenting and he said he'd prefer you didn't?